### PR TITLE
Release 2.20 updates

### PR DESCRIPTION
**Pull request by dsarkar**
- [x] Update gulpfile.js 2.20
- [x] Update release date
- [x] Privacy notice release 2.20 --> **NO UPDATES**
https://github.com/corona-warn-app/cwa-website/pull/2542
- [x] Screenshots
https://github.com/corona-warn-app/cwa-website/pull/2543
- [x] FAQ
  - Hotline https://github.com/corona-warn-app/cwa-website/issues/2539#issuecomment-1071003841
https://github.com/corona-warn-app/cwa-website/pull/2597 (will be merged separately) 
  - https://github.com/corona-warn-app/cwa-website/pull/2595 
  -  https://github.com/corona-warn-app/cwa-website/pull/2615 
  - https://github.com/corona-warn-app/cwa-website/pull/2619  
  - https://github.com/corona-warn-app/cwa-website/pull/2620
  - https://github.com/corona-warn-app/cwa-website/pull/2627
  - https://github.com/corona-warn-app/cwa-website/pull/2628
  - https://github.com/corona-warn-app/cwa-website/pull/2630
  - https://github.com/corona-warn-app/cwa-website/pull/2632
- [x] Blog

---
**Release 2.20**
- Blog
  - DE https://www.coronawarn.app/de/blog/2022-03-30-cwa-2-20/
  - EN https://www.coronawarn.app/en/blog/2022-03-30-cwa-2-20/
- Screenshots
  - DE https://www.coronawarn.app/de/screenshots/ 
  - EN  https://www.coronawarn.app/en/screenshots/
- Privacy notice was not updated. Actual version is R 2.19.
  - DE https://www.coronawarn.app/de/privacy/
  - EN https://www.coronawarn.app/en/privacy/ 
- FAQ (new or updated)
  - DE **Werden Genesenenzertifikate unterstützt?**
https://www.coronawarn.app/de/faq/#rec_cert 
  - EN **Are certificates of recovery supported?**
https://www.coronawarn.app/en/faq/#rec_cert 
- FAQ (new or updated)
  - DE **Grundlegendes zur Anzeige des Status-Nachweises**
https://www.coronawarn.app/de/faq/#admission_policy
  - EN **General information about the status proof display**
https://www.coronawarn.app/en/faq/#admission_policy
- FAQ (new or updated)
  - DE **Werden zu einer Risiko-Begegnung Personen und/oder Orte zugeordnet?**
https://www.coronawarn.app/de/faq/#risk_encounter_place
  - EN **Are people and/or places assigned to a risk encounter?**
https://www.coronawarn.app/en/faq/#risk_encounter_place
- FAQ (new or updated)
  - DE **Wie oft wird der Risiko-Status aktualisiert?**
https://www.coronawarn.app/de/faq/#risk_update_24
  - EN **How often is the risk status updated?**
https://www.coronawarn.app/en/faq/#risk_update_24
- FAQ -- add Ukrainian to #available_languages
  - DE **In welchen Sprachen gibt es die Corona-Warn-App?**
https://www.coronawarn.app/de/faq/#available_languages
  - EN **In which languages is the Corona-Warn-App available?**
https://www.coronawarn.app/en/faq/#available_languages
- FAQ dcc_replacement
  - DE Können Zertifikate über die App aktualisieren werden ? 
https://www.coronawarn.app/de/faq/#dcc_replacement
  - EN Can certificates be updated via the app?
https://www.coronawarn.app/en/faq/#dcc_replacement
- Multiple FAQ articles were updated regarding the change from 14 to 10 days.

   
**Other new or updated content (Not Release 2.20 related)**

- **Gebärdensprache** https://www.coronawarn.app/de/sign-language/
- **Leichte Sprache** https://www.coronawarn.app/de/simple-language/
- New article #delta_submission
  - DE **Warum werden seit Version 2.19 weniger Schlüssel in den Begegnungsaufzeichnungen abgeglichen?**
https://www.coronawarn.app/de/faq/#delta_submission
  - EN **Why are fewer keys matched since version 2.19?**
https://www.coronawarn.app/en/faq/#delta_submission
- Updated #personal_data: Replaced the outdated list of data in the  FAQ article with a link to the privacy policy in accordance with the details from #1729 .
  - DE: **Welche personenbezogenen Daten werden in der App gespeichert?**
https://www.coronawarn.app/de/faq/#personal_data
  - EN: **What personal data is stored in the app?**
https://www.coronawarn.app/en/faq/#personal_data
- Update technical hotline times #international_phone_numbers (to be merged later on)
  - DE **So erreichen Sie uns**
https://www.coronawarn.app/de/faq/#international_phone_numbers
  - EN **How to reach us**
https://www.coronawarn.app/en/faq/#international_phone_numbers

---
Internal Tracking ID: [EXPOSUREAPP-11957](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11957)

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -491,10 +491,10 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + "/**/*.html"])
     .pipe(replace('[ios.latest-os-version]', '15.4'))
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
-    .pipe(replace('[ios.current-app-version]', '2.19.1'))
+    .pipe(replace('[ios.current-app-version]', '2.20.0'))
     .pipe(replace('[android.latest-os-version]', '12'))
     .pipe(replace('[android.minimum-required-os-version]', '6'))
-    .pipe(replace('[android.current-app-version]', '2.19.2'))
+    .pipe(replace('[android.current-app-version]', '2.20.0'))
```
Comment (Ein-Tim) [inline gulpfile.js:497]:
```suggestion
    .pipe(replace('[android.current-app-version]', '2.20.1'))
```


Will probably be 2.20.1, see latest RC: https://github.com/corona-warn-app/cwa-app-android/releases/tag/v2.20.1-rc.0

File: gulpfile.js [hunk 2]
```diff
@@ -491,10 +491,10 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + "/**/*.html"])
     .pipe(replace('[ios.latest-os-version]', '15.4'))
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
-    .pipe(replace('[ios.current-app-version]', '2.19.1'))
+    .pipe(replace('[ios.current-app-version]', '2.20.0'))
```
Comment (Ein-Tim) [inline gulpfile.js:494]:
```suggestion
    .pipe(replace('[ios.current-app-version]', '2.20.1'))
```

Will probably be 2.20.1, see latest RC: https://github.com/corona-warn-app/cwa-app-ios/releases/tag/v2.20.0-rc.5

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
Bump version

**Comment by Ein-Tim** [comment]
Does this PR address all instances where the website states that the app considers the last 14 days?

File: gulpfile.js (hunk 1 above)
Comment (dsarkar) [reply gulpfile.js:497]: thank @Ein-Tim

File: gulpfile.js (hunk 2 above)
Comment (dsarkar) [reply gulpfile.js:494]: thanks again @Ein-Tim

**Pull request merged**